python/src/trezorlib/messages.py

```
@@ -4602,6 +4627,122 @@ def __init__(
         self.name = name
 
 
+class EthereumABITupleInfo(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = None
+    FIELDS = {
+        1: protobuf.Field("fields", "EthereumABIType", repeated=True, required=False, default=None),
+        2: protobuf.Field("is_dynamic", "bool", repeated=False, required=True),
+    }
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)